### PR TITLE
Devise setup - pending specs

### PR DESCRIPTION
**Pull request by maebeale**

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  # If true, extends the user's remember period when remembered via cookie.
+  # config.extend_remember_period = false
+
+  # Options to be passed to the created cookie. For instance, you can set
+  # secure: true in order to force SSL only cookies.
+  # config.rememberable_options = {}
+
+  # ==> Configuration for :validatable
+  # Range for password length.
+  config.password_length = 6..128
```
Comment (exbinary) [inline config/initializers/devise.rb]: Maybe bump this up to 8?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+  # Defines which strategy will be used to unlock an account.
+  # :email = Sends an unlock link to the user email
+  # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
+  # :both  = Enables both strategies
+  # :none  = No unlock strategy. You should handle unlocking by yourself.
+  # config.unlock_strategy = :both
+
+  # Number of authentication tries before locking an account if lock_strategy
+  # is failed attempts.
+  # config.maximum_attempts = 20
```
Comment (exbinary) [inline config/initializers/devise.rb]: Is 20 too many? I don't have an opinion on this, just wondering.

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+  #     mount MyEngine, at: '/my_engine'
+  #
+  # The router that invoked `devise_for`, in the example above, would be:
+  # config.router_name = :my_engine
+  #
+  # When using OmniAuth, Devise cannot automatically set OmniAuth path,
+  # so you need to do it manually. For the users scope, it would be:
+  # config.omniauth_path_prefix = '/my_engine/users/auth'
+
+  # ==> Turbolinks configuration
```
Comment (exbinary) [inline config/initializers/devise.rb]: I think turbolinks might be enabled by default in stock rails 6?

**Comment by maebeale** [comment]
Yup, exactly re admin. I think we could start with a boolean and go from
there.

File: app/views/layouts/_notices_and_alerts.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<% if notice %>
+  <div class="alert alert-info alert-dismissible text-center short-response" role="alert">
+    <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```
[the remark this answers is not in the document]
Comment (maebeale) [reply app/views/layouts/_notices_and_alerts.html.erb]: correct

File: config/initializers/devise.rb (hunk 2 above)
Comment (maebeale) [reply config/initializers/devise.rb]: yeah, when this is turned on i usually have it at 3 or 5. will uncomment this so at least 20 is on. do you have a preferred/recommended number?

File: config/initializers/devise.rb (hunk 3 above)
Comment (maebeale) [reply config/initializers/devise.rb]: YES! THANKS for catching that!

File: config/initializers/devise.rb (hunk 1 above)
Comment (maebeale) [reply config/initializers/devise.rb]: k

File: config/initializers/devise.rb (hunk 2 above)
Comment (exbinary) [reply config/initializers/devise.rb]: 5 sounds good.

**Pull request merged**